website/content/en/status/report-2023-04-2023-06/freebsd-foundation.adoc

```diff
@@ -0,0 +1,165 @@
+
+==== Partnership Program
+
+Hello FreeBSD community.
+I'm Greg Wallace.
+I joined the Foundation as Director of Partnerships and Research in early April.
+link:https://freebsdfoundation.org/blog/freebsd-foundation-welcomes-new-team-members/[This blog introduced me and the role].
+For Partnerships, I am focused on building connections with companies that use FreeBSD.
+I have met with several companies to learn about how they use FreeBSD.
+Some of these meetings have generated discussions about potential partnership.
+I continue to find out about interesting companies using FreeBSD and am reaching out to them.
+
+My objective is to get in touch with every company building with and using FreeBSD to listen to their stories.
+If this is you and we have not yet connected, please schedule a call on my calendar with link:https://calendly.com/greg-freebsdfound/30min[this link].
+
+Some other partnership-related activities this Quarter:
+
+* I created link:https://docs.google.com/presentation/d/1tDCpbfxbqIucmJF6H15vK-ETrQsCMOVtxoqLem_V0Z0/edit?usp=sharing[these slides] about how partnering with the Foundation helps advance FreeBSD.
+  If you have ideas for how I can improve these slides, or would like me to present them to your organization, please link:mailto:greg@freebsdfoundation.org[shoot me an email], or link:https://calendly.com/greg-freebsdfound/30min[schedule a call].
+  And please feel free to share the presentation liberally, in whole or in part.
+* I worked with my Foundation colleagues to create a number of industry-specific use case slides for a presentation to an industry analyst.
+* I am also pursuing Grant opportunities with bodies including:
+** NSF Secure and Trustworthy Cyberspace (SaTC)
+** Sovereign Tech Fund
+** NGI
+
+In terms of Research, my broad aim is to make sure that all of the expertise in this community is reflected in the global conversations on computing performance, security, and energy efficiency.
+As a community, we have much to bring to this work.
+
+So far, I have been tracking and plugging into the following threads:
+
+* link:https://openforumeurope.org/open-source/[Open Forum Europe]
+* link:https://www.nist.gov/chips/research-and-development-program[CHIPS Research and Development]
+
+If you have research ideas or are interested in working together in this area, please link:mailto:greg@freebsdfoundation.org[shoot me an email], or link:https://calendly.com/greg-freebsdfound/30min[schedule a call].
+
+==== OS Improvements
+
+During the second quarter of 2023, 339 src, 155 ports, and 20 doc tree commits identified the FreeBSD Foundation as a sponsor.
+Some of this and other Foundation-sponsored work is described in separate report entries:
+
+* Continuous Integration
+* FreeBSD as a Tier 1 cloud-init Platform
+* OpenSSL 3 in base
+* OpenStack on FreeBSD
+* Security Sandboxing Using man:ktrace[1]
+* SIMD enhancements for amd64
+
+Here is a sampling of other Foundation-sponsored work:
+
```

Comment:
Consistency: if you used capital letters for the precedent list, please use them here too.